lib/davtypes.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of buteo-sync-plugin-caldav package
+ *
+ * Copyright (C) 2025 Damien Caliste <dcaliste@free.fr>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA
+ */
+
+#ifndef DAVTYPES_H
+#define DAVTYPES_H
+
+#include <QString>
+#include <QList>
+#include <QFlags>
+
+#include "davexport.h"
+
+namespace Buteo {
+namespace Dav {
+enum DAV_EXPORT Privilege {
```

Comment:
Since enums is mostly a shorthand for naming numeric values, I wouldn't expect the export here to do much.

Reply:
Ah right, the header is enough. There is nothing to put in the .so. Removed.